api4/user_test.go

```diff
@@ -2806,6 +2806,26 @@ func TestUpdateUserPassword(t *testing.T) {
 	CheckNoError(t, resp)
 }
 
+func TestUpdateUserHashedPassword(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	password := "SuperSecurePass23!"
+	passwordHash := "$2a$10$CiS1iWVPUj7rQNdY6XW53.DmaPLsETIvmW2p0asp4Dqpofs10UL5W"
+	pass, resp := th.SystemAdminClient.UpdateUserHashedPassword(th.BasicUser.Id, passwordHash)
```

Comment:
this test can be wrapped using the `th.TestForSystemAdminAndLocal` helper so it checks local mode too

Reply:
Awesome, thanks for the tip!